Samples/java/src/main/java/com/phunware/java/sample/BluedotLocationActivity.java

```diff
@@ -26,6 +26,8 @@ a copy of this software and associated documentation files (the
 other dealings in this Software without prior written authorization
 from Phunware, Inc. */
 
+import static com.phunware.java.sample.CustomMapFragment.getInstance;
```

Comment:
Nitpick: Static import of the `getInstance()` method makes the code a little hard to read when we are doing our fragment transaction, as it's not immediately apparent on line 84 as to what fragment we are replacing in the layout.